avr/cores/megacommand/MCL/SeqExtStepPage.cpp

```diff
@@ -589,7 +589,7 @@ bool SeqExtStepPage::handleEvent(gui_event_t *event) {
     uint8_t track = event->source - 128;
 
     if (device == DEVICE_A4) {
-      track -= 16;
+      return true;
```

Comment:
this broke your A4. @yatli

Reply:
ignore... UART2 used for MIDI notes ?

Reply:
Nope, that was an effort trying to make it work.

Reply:
I'm wondering why A4 note events don't show up in midi2 callbacks.
Keyboard CFG=Ext

Reply:
it's working for me...

Reply:
new firmware? keyboard needs to be set to transmit notes.

Reply:
It works in the step edit page, triggers MD notes

Reply:
eh, my bad. auto track channel was set to 9

Reply:
MIDI newbie :-p

Reply:
the correct config should be, midi out ch -> TRK; was set to auto.

Reply:
That reminds me, we need channel selection in ext track menu.